src/main.js

```diff
@@ -3,17 +3,25 @@ import 'wow.js/css/libs/animate.css';
 import '@fortawesome/fontawesome-free/css/all.min.css';
 import 'swiper/css/bundle';
 
+import advantages from '../assets/advantages-image.png'
+import questions from '../assets/questions-image.png'
+import pizza from '../assets/pizza.png'
+import burger from '../assets/burger.png'
+import sushi from '../assets/sushi.png'
+import cookies from '../assets/cookies.png'
+import puncake from '../assets/puncake.png'
```

Comment:
please, always provide semi-columns on the end of the line (keep code clean, check other places for possible improvement)